src/main/plugins/websearch-plugin/websearch-plugin.ts

```diff
@@ -22,7 +23,8 @@ export class WebSearchPlugin implements ExecutionPlugin {
 
     public getSearchResults(userInput: string, fallback?: boolean): Promise<SearchResultItem[]> {
         return new Promise((resolve, reject) => {
-            const searchResults = this.config.webSearchEngines
+            const searchResults: SearchResultItem[] = [];
+            const webSearchEnginesArray = this.config.webSearchEngines
```

Comment:
In my opinion this variable could be called `webSearchEngines` instead of `webSearchEnginesArray`. This should be self self-explanatory and the IDE can tell you of what type the variable is.

Reply:
I will do that.